mrjob/tools/emr/terminate_idle_job_flows.py

```diff
@@ -93,11 +109,23 @@ def main(cl_args=None):
         pooled_only=options.pooled_only,
         max_mins_locked=options.max_mins_locked,
         quiet=(options.quiet > 1),
+        **runner_kwargs(options)
     )
 
 
-def inspect_and_maybe_terminate_job_flows(
-    conf_paths=None,
+def runner_kwargs(options):
+    kwargs = options.__dict__.copy()
+    for unused_arg in ('quiet', 'verbose', 'max_hours_idle',
+                       'max_mins_locked',
+                       'mins_to_end_of_hour', 'unpooled_only',
+                       'pooled_only', 'pool_name', 'dry_run',
+                       'test'):
+        del kwargs[unused_arg]
```

Comment:
This starts to be a pretty big list - how maintainable is this? Would it be better to switch to a whitelist instead of a blacklist?